server/src/main/java/org/sonarsource/ldap/server/ApacheDS.java

```diff
@@ -176,6 +186,39 @@ private ApacheDS startLdapServer() throws Exception {
     return this;
   }
 
+  private ApacheDS startKdcServer() throws Exception {
+    int port = AvailablePortFinder.getNextAvailable(6088);
+
+    KerberosConfig kdcConfig = new KerberosConfig();
+    kdcConfig.setServicePrincipal("krbtgt/EXAMPLE.ORG@EXAMPLE.ORG");
+    kdcConfig.setPrimaryRealm("EXAMPLE.ORG");
+    kdcConfig.setMaximumTicketLifetime(86400000L);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Add underscores to this numeric value for readability [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2148)